Content.Client/Flight/FlightSystem.cs

```diff
@@ -0,0 +1,75 @@
+using Robust.Client.GameObjects;
+using Content.Shared.Flight;
+using Content.Shared.Flight.Events;
+using Content.Client.Flight.Components;
+
+namespace Content.Client.Flight
+{
```

Comment:
```suggestion
namespace Content.Client.Flight;
```
You are *also* going to need to move the entire document down an indent after doing this. Ctrl-A and Shift-Tab to do the entire document all at once.